docs/guide/ecosystem/server-side-rendering.md

```diff
@@ -1,22 +1,110 @@
 # Server Side Rendering
 
-## When and why to use SSR
+Server side rendering can be explained as moving the process of rendering pages of an SPA from the client's browser to the server. This process is similar to what we used to do a few years ago, where we had server side languages like PHP, Java, Python render the page and return an HTML response.
+
+The difference is, that once the page gets loaded by the client's browser, the JavaScript framework, Vue in our case, takes over the application's routing and rendering processes, giving the impression of a faster navigation and smoother experience.
```

Comment:
I'd highlight that SSR only happens on the **initial** request. That's what most ppl, including me, get wrong when starting with that topic.

Reply:
Dang I forgot to mention it. Good job!